juicefs panic in redis engine when inode is corrupted

The #3266 fix #3263, in which case the inode is lost. However, in another case that inode is corrupted (but still exist), juicefs will also panic.

backtrace:

```log
panic: runtime error: slice bounds out of range [:19] with capacity 16

goroutine 1 [running]:
github.com/juicedata/juicefs/pkg/utils.(*Buffer).Get64(...)
        /home/hexi/src/juicefs/pkg/utils/buffer.go:118
github.com/juicedata/juicefs/pkg/meta.(*baseMeta).parseAttr(0x0?, {0xc000ad8060, 0xe, 0x10}, 0xc002cb5ef0)
        /home/hexi/src/juicefs/pkg/meta/base.go:576 +0x7df
github.com/juicedata/juicefs/pkg/meta.(*redisMeta).doLookup(0xc0002af270, {0x3286bc0?, 0xc0035a61c0}, 0xc0002bc0f0?, {0x47294b8, 0x1}, 0xc0020662c0, 0xc002cb5ef0)
        /home/hexi/src/juicefs/pkg/meta/redis.go:666 +0x6e9
github.com/juicedata/juicefs/pkg/meta.(*baseMeta).Lookup(0xc000079400, {0x3286bc0, 0xc0035a61c0}, 0x66ee8, {0x47294b8, 0x1}, 0xc0020662c0, 0xc002cb5ef0)
        /home/hexi/src/juicefs/pkg/meta/base.go:551 +0x446
github.com/juicedata/juicefs/pkg/vfs.(*VFS).Lookup(0xc0006e6000, {0x3287820?, 0xc0035a61c0}, 0x66ee8, {0x47294b8, 0x1})
        /home/hexi/src/juicefs/pkg/vfs/vfs.go:93 +0x265
github.com/juicedata/juicefs/pkg/fuse.(*fileSystem).Lookup(0xc000482c80, 0x235cf59?, 0xc00082c3d8, {0x47294b8, 0x1}, 0xc00082c348?)
        /home/hexi/src/juicefs/pkg/fuse/fuse.go:85 +0xc5
github.com/hanwen/go-fuse/v2/fuse.doLookup(0xc00082c240?, 0xc00082c240)
        /home/hexi/go/pkg/mod/github.com/juicedata/go-fuse/v2@v2.1.1-0.20221212033224-ed3172586347/fuse/opcode.go:342 +0x69
github.com/hanwen/go-fuse/v2/fuse.(*Server).handleRequest(0xc00081e160, 0xc00082c240)
        /home/hexi/go/pkg/mod/github.com/juicedata/go-fuse/v2@v2.1.1-0.20221212033224-ed3172586347/fuse/server.go:492 +0x1f3
github.com/hanwen/go-fuse/v2/fuse.(*Server).loop(0xc00081e160, 0xc8?)
        /home/hexi/go/pkg/mod/github.com/juicedata/go-fuse/v2@v2.1.1-0.20221212033224-ed3172586347/fuse/server.go:465 +0x108
github.com/hanwen/go-fuse/v2/fuse.(*Server).Serve(0xc00081e160)
        /home/hexi/go/pkg/mod/github.com/juicedata/go-fuse/v2@v2.1.1-0.20221212033224-ed3172586347/fuse/server.go:387 +0x35
github.com/juicedata/juicefs/pkg/fuse.Serve(0xc0006e6000, {0x0, 0x0}, 0x0)
        /home/hexi/src/juicefs/pkg/fuse/fuse.go:473 +0x5fe
github.com/juicedata/juicefs/cmd.mount_main(0xc0006e6000, 0x329e8d0?)
        /home/hexi/src/juicefs/cmd/mount_unix.go:182 +0x23e
github.com/juicedata/juicefs/cmd.mount(0xc000a8b1c0)
        /home/hexi/src/juicefs/cmd/mount.go:454 +0x7a8
github.com/urfave/cli/v2.(*Command).Run(0xc00083d7a0, 0xc000a8a740)
        /home/hexi/go/pkg/mod/github.com/urfave/cli/v2@v2.4.0/command.go:163 +0x5c8
github.com/urfave/cli/v2.(*App).RunContext(0xc000602820, {0x3273330?, 0xc0000560a0}, {0xc000a8a200, 0x4, 0x4})
        /home/hexi/go/pkg/mod/github.com/urfave/cli/v2@v2.4.0/app.go:313 +0xb36
github.com/urfave/cli/v2.(*App).Run(...)
        /home/hexi/go/pkg/mod/github.com/urfave/cli/v2@v2.4.0/app.go:224
github.com/juicedata/juicefs/cmd.Main({0xc000a8a080?, 0x4?, 0x4?})
        /home/hexi/src/juicefs/cmd/main.go:86 +0x258d
main.main()
        /home/hexi/src/juicefs/main.go:29 +0x2e
```

--- a/go.mod
+++ b/go.mod
@@ -242,7 +242,7 @@ require (
 
 replace github.com/minio/minio v0.0.0-20210206053228-97fe57bba92c => github.com/juicedata/minio v0.0.0-20221113011458-8866d5c9df8c
 
-replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20221212033224-ed3172586347
+replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20230417143528-b35fdce8e2b1
 
 replace github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt v3.2.1+incompatible
 

--- a/go.mod
+++ b/go.mod
@@ -242,7 +242,7 @@ require (
 
 replace github.com/minio/minio v0.0.0-20210206053228-97fe57bba92c => github.com/juicedata/minio v0.0.0-20221113011458-8866d5c9df8c
 
-replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20230417143528-b35fdce8e2b1
+replace github.com/hanwen/go-fuse/v2 v2.1.1-0.20210611132105-24a1dfe6b4f8 => github.com/juicedata/go-fuse/v2 v2.1.1-0.20230418090413-880e994b8a4c
 
 replace github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt v3.2.1+incompatible
 

